eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Ratify.hs

```diff
@@ -217,11 +213,13 @@ ratifyTransition = do
                   }
           trans @(ConwayRATIFY era) $ TRC (env, st', RatifySignal sigs)
         else do
+          -- This action hasn't been ratified yet. Process the remaining actions.
           st' <- trans @(ConwayRATIFY era) $ TRC (env, st, RatifySignal sigs)
-          if expired
-            then -- Action expired, do not include it in the next epoch
+          -- Finally, filter out actions that are not processed.
+          if gasExpiresAfter >= reCurrentEpoch
```

Comment:
Wait, so if the currentEpoch is smaller than the expiration date, this means that it's expired?  Is this right?

Reply:
You are absolutely right :smile: Updated my suggestion.

Reply:
Thanks for the awesome catch :raised_hands: